### ISSUE TITLE
README.md contains train.py, but it is not used and there is no explanation what that is

### ISSUE THREAD

**Issue by rualark** [pytorch/torchft#161]
See here: https://github.com/pytorch/torchft/tree/main?tab=readme-ov-file#example-training-loop-ddp

train.py:

from torchft import Manager, DistributedDataParallel, Optimizer, ProcessGroupGloo

manager = Manager(
    pg=ProcessGroupGloo(),
    load_state_dict=...,
    state_dict=...,
)

m = nn.Linear(2, 3)
m = DistributedDataParallel(manager, m)
optimizer = Optimizer(manager, optim.AdamW(m.parameters()))

for i in range(1000):
    batch = torch.rand(2, 2, device=device)

    optimizer.zero_grad()

    out = m(batch)
    loss = out.sum()

    loss.backward()

    optimizer.step()

**Comment by rualark** [comment]
It seems that this whole train.py section should be removed from README.md or it should be explained why it is partially duplicating train_ddp.py?

**Comment by d4l3k** [comment]
@rualark `train.py` in the README isn't referring to a specific file -- just a general high level API reference for context. We could remove it if you don't think it provides any value there

**Comment by rualark** [comment]
It seems that it could provide value if explained what it is in the doc. Right not it seems to cause more confusion and duplication, and probably removing may be a better solution.
